lib/main.dart

```diff
@@ -105,7 +105,11 @@ class _RandomWordsState extends State<RandomWords> {
             appBar: AppBar(
               title: const Text('Saved Suggestions'),
             ),
-            body: ListView(children: divided),
+            body: divided.isNotEmpty
+                ? ListView(children: divided)
+                : const Center(
```

Comment:
Good decision to use `const ` as it's an optimization for your app; your object becomes immutable.